public/js/bracketlogic.js

```diff
@@ -0,0 +1,188 @@
+//16.A pick
```

Comment:
These functions are fairly similar, it looks like you could probably write all of these with a for loop and some small changes to your code. Try to figure it out otherwise come talk to us and we can help you out.

Reply:
always remember DRY!